services/frontend/src/scenes/Profile/connect.js

```diff
@@ -3,8 +3,9 @@ import {crossSliceSelectors} from "store/";
 import {profilesRequestsSlice} from "store/slices";
 
 const mapStateToProps = (state) => ({
-    userProfile: crossSliceSelectors.getUserProfile(state),
-    profileLoadingState: profilesRequestsSlice.fetchAll.selectors.getLoading(state)
+    profileLoadingState: profilesRequestsSlice.fetchAll.selectors.getLoading(state),
+    projects: crossSliceSelectors.getProjectsForUser(state),
```

Comment:
Should we keep these variables named consistently as `userProjects` and `userProfile`? (or `profile`/`projects`)

Probably don't need the loading state to be differently named though since that's just the profile as a page.

Reply:
I'd say it they shouldn't even have `user` prepended to them; it should just be `profile`, `projects`, and `isLoading`. The `user` doesn't really add any extra information to the prop; in fact, once we start using this `Profile` component for the non-logged in user profiles, it makes _no sense_ to put `user` in front, since it's no longer the 'logged-in' user's profile.

Reply:
Yep, I would think it only needs `user` prepended if we are going to be mixing the current users data with other users data but as far as I can tell we aren't going to be doing anything like that on any page of our MVP.

Reply:
Then since you agree, I'm going to make those naming changes now since we all know that it'll be forgotten about later.